index.html

```diff
@@ -4258,6 +4259,23 @@ <h2 id="MediaElementAudioSourceNode">
 </pre>
         <section>
           <h2>
+            MediaElementAudioSourceOptions
+          </h2>
+          <p>
+            This specifies the options to use in constructing a
+            <a><code>MediaElementAudioSourceNode</code></a>. This is required.
+          </p>
+          <dl title="dictionary MediaElementAudioSourceOptions" class="idl">
+            <dt>
```

Comment:
Good question. I don't know. My guess is that if there there might be
future options, but for an AudioNodeOptions object, it really needs to be
fairly generic to apply to every node, so it wouldn't be appropriate to a
MediaElementAudioSourceNode.

On Sun, Jul 31, 2016 at 9:35 AM, Joe Berkovitz <notifications@github.com>
wrote:

> In index.html
> <https://github.com/WebAudio/web-audio-api/pull/897#discussion_r72906932>:
>
> > @@ -4258,6 +4259,23 @@ <h2 id="MediaElementAudioSourceNode">
> >  </pre>
> >          <section>
> >            <h2>
> > +            MediaElementAudioSourceOptions
> > +          </h2>
> > +          <p>
> > +            This specifies the options to use in constructing a
> > +            <a><code>MediaElementAudioSourceNode</code></a>. This is required.
> > +          </p>
> > +          <dl title="dictionary MediaElementAudioSourceOptions" class="idl">
> > +            <dt>
>
> Should this derive from AudioNodeOptions? Do we know in advance that no
> future options added to AudioNode can affect this node?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WebAudio/web-audio-api/pull/897/files/3314d43c88bc21903194e82a691f47b9c3307ad7#r72906932>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAofPMnVZlOmggRovMTv9gzSI7DT8vMEks5qbM7RgaJpZM4JHX7q>
> .
>



-- 
Ray